ITEM.tsv

```diff
@@ -10,9 +10,9 @@ ITEM_20150317_000009	$Tasseled Armband
 ITEM_20150317_000010	$Empty_weapon 
 ITEM_20150317_000011	{memo X}$Crude Gladius 
 ITEM_20150317_000012	$One-Handed Sword [Slash] 
-ITEM_20150317_000013	$A weapon that is commonly used and carried by soldiers of the kingdom. 
+ITEM_20150317_000013	$A weapon that is commonly used and carried by soldiers.
```

Comment:
It could even be shortened to:
A weapon that is commonly used by the kingdom's soldiers. :astonished: